robottelo/common/helpers.py

```diff
@@ -117,6 +123,27 @@ def get_server_cert_rpm_url():
         get_server_pub_url(), 'katello-ca-consumer-latest.noarch.rpm')
 
 
+def get_host_info(hostname=None):
+    """Get remote host's distribution information
+
+    :param str hostname: Hostname or IP address of the remote host. If ``None``
+        the hostname will be get from ``main.server.hostname`` config.
+
+    """
+    result = ssh.command('cat /etc/redhat-release', hostname)
+    if result.return_code != 0:
+        raise HostInfoError('/etc/redhat-release not found')
```

Comment:
There are numerous reasons why the ssh.command might fail, wouldn't it be better to pass `result.stderr` as the exception message to be more helpful?